common/src/main/java/com/microsoft/identity/common/exception/ServiceException.java

```diff
@@ -139,6 +156,7 @@ public void setHttpResponse(final HttpResponse response) throws JSONException {
     public ServiceException(final String errorCode, final String errorMessage, final Throwable throwable) {
         super(errorCode, errorMessage, throwable);
         mHttpStatusCode = DEFAULT_STATUS_CODE;
+        mSubErrorCode = "";
```

Comment:
Any reason we are initializing here to empty string , can't just be left null ?

Reply:
I could leave it null yes.